src/GitHub.Exports/Services/VSServices.cs

```diff
@@ -15,11 +17,13 @@ namespace GitHub.Services
     public class VSServices : IVSServices
     {
         readonly IGitHubServiceProvider serviceProvider;
+        readonly DTE dte = null;
 
         [ImportingConstructor]
-        public VSServices(IGitHubServiceProvider serviceProvider)
+        public VSServices(IGitHubServiceProvider serviceProvider, SVsServiceProvider vsServiceProvider)
```

Comment:
This will make this class hard to mock and bound to specific VS types, and you shouldn't need to do this, the `IGitHubServiceProvider` service should be able to give you what you need (it aggregates services from different types and different service and MEF look ups all in one place, so you don't have to try and guess which service contains which thing).